testing/test_unittest.py

```diff
@@ -537,28 +537,24 @@ def f(_):
         )
         result.stdout.fnmatch_lines(
             [
-                "test_trial_error.py::TC::test_four SKIPPED",
+                "test_trial_error.py::TC::test_four FAILED",
```

Comment:
This reverts the test to the state before 04f27d4, which introduced the breaking change about addCleanup not being called properly for failed tests.